index.html

```diff
@@ -301,17 +330,19 @@ <h2 class="top-heading">About &nbsp; &nbsp; &nbsp; &nbsp; Myself</h2>
               <span>Languages</span>
               <img class="skill-img1" src="./images/Union.png" alt="arrow" />
             </div>
-            <div class="javascript1">
-              <img src="./images/Ellipse 1.png" alt="js" />
-              <p>JavaScript</p>
-            </div>
-            <div class="html1">
-              <img src="./images/Ellipse 1 (1).png" alt="html" />
-              <p>HTML</p>
-            </div>
-            <div class="css1">
-              <img src="./images/Ellipse 1 (2).png" alt="css" />
-              <p>CSS</p>
+            <div class="Languages-desktop">
+              <div class="javascript1">
+                <img src="./images/Ellipse 1.png" alt="js" />
```

Comment:
It's super nice 👌but you can make it even better by adding some left padding to the languages icon box 😊

![image](https://user-images.githubusercontent.com/34779928/194422797-df24016e-ad4a-4e09-8e5a-51723462ebaf.png)